include/git2/blob.h

```diff
@@ -108,6 +108,12 @@ typedef enum {
 	 * system-wide `gitattributes` in `/etc` (or system equivalent).
```

Comment:
There's a word missing in the commit message here: "When `GIT_BLOB_FILTER_ATTTRIBUTES_FROM_HEAD` to `git_blob_filter`". Probably "passing"?